provider/marathon/marathon_test.go

```diff
@@ -522,18 +522,42 @@ func TestMarathonApplicationFilterConstraints(t *testing.T) {
 		desc                    string
 		application             marathon.Application
 		marathonLBCompatibility bool
+		marathonConstraints     bool
 		expected                bool
 	}{
 		{
 			desc:                    "tags missing",
 			application:             application(),
 			marathonLBCompatibility: false,
+			marathonConstraints:     false,
```

Comment:
Should we omit `marathonConstraint` (and possibly other parameters) when they are not needed / relevant for a test?